flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -46,107 +45,93 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
             autoScalerConfig("enabled")
                     .booleanType()
                     .defaultValue(false)
-                    .withFallbackKeys(oldOperatorConfigKey("enabled"))
```

Comment:
Do we need to remove it? I'm not sure should we keep it to let old users know it.

Reply:
The core idea of this fix is to remove deprecated/fallback keys because they generate WARN messages in the logs for every key and every reconciliation loop. The keys are deprecated in the sense that they are not documented anymore but we can't remove them, at least not until much later.

Reply:
After migration, the log already can be removed, right?

Reply:
Yes, after migration the fallback keys are not needed anymore.